anda/terra/gpg-keys/terra-gpg-keys.spec

```
@@ -0,0 +1,55 @@
+Name:           terra-gpg-keys
+Version:        1
```

Technically should be the Fedora release ver so we can do this funny.
```suggestion
%undefine dist

Name:           terra-gpg-keys
Version:        %{?fedora:%{fedora}}%{?rhel:%{rhel}}
```